### DISCUSSION TITLE
Lock playlist feature

### DISCUSSION THREAD

**Discussion by Fooftilly** [ReVanced/revanced-discussions#141] [Question]
Would it be possible to add a feature where you can lock a playlist so it doesn't show in "add to playlist" dialog? This would be useful to people like me who have a lot of playlists and can accidentally add videos to wrong playlist.

**Comment by MSOB7YY** [comment]
I WOULD DEFINITELY WANT THAT
also locking playlists from sorting or removing videos inside 🤤

**Comment by oSumAtrIX** [comment]
I don't understand the feature request, what do you mean by locking?

**Comment by Fooftilly** [comment]
Example of what I meant:
Let's say that I have 10 playlists created. When I open some video and hold "add to playlist" button, Youtube will show me all those playlist, and I will be able to choose to which one I would like to add the video. I would like to be able to have a filter where you can select which playlists you wouldn't want to be shown when adding videos to the playlists. So in short I would be able to choose playlist 1-7, and only playlist 8,9 and 10 would be shown, or any of the playlists which are not in the filter list.

**Comment by oSumAtrIX** [comment]
Sounds really specific, not sure if people want this

**Comment by TwilightSublime** [comment]
I want this feature as well

**Comment by Berny23** [comment]
+1 I need this, because I have some more or less "static" public playlists I don't want to change accidentally. On PC, this isn't as big of a problem without touch controls and holding down an icon.
